src/lib/MongoQueue.ts

```diff
@@ -0,0 +1,76 @@
+import { Model, Schema } from 'mongoose';
+import { Queue, Media, QueueManager } from 'musicca';
+
+export interface IQueue {
+  _id: string;
+  list: Media[];
+}
+
+export const schema = new Schema<IQueue>({
+  _id: String,
+  list: {
+    type: [Object],
+    default: []
+  }
+});
+
+export default class MongoQueue extends Queue {
+  public db: Model<IQueue>;
+
+  public id: string;
+
+  constructor(model: Model<IQueue>, manager: QueueManager<Queue>, id: string) {
```

Comment:
Don't put `model` as the first argument. I've added the ability to add options ([commit here](https://github.com/musiccajs/musicca/commit/e18a34005f5bb6f00a3f74ca9a88af0655c0ef9f)), please use that instead.